packages/live-status-gateway/api/schemas/adLibs.yaml

```diff
@@ -60,17 +60,64 @@ $defs:
               type: string
           required: [name, label]
           additionalProperties: false
-      tags:
-        description: Tags attached to this AdLib
-        type: array
-        items:
-          type: string
       publicData:
         description: Optional arbitrary data
     required: [id, name, sourceLayer, actionType]
     additionalProperties: false
+  adLib:
+    allOf:
+      - $ref: '#/$defs/baseAdLib'
+      - type: object
+        properties:
+          partId:
+            description: Unique id of the part which owns this AdLib
+            type: string
+    examples:
+      - id: 'C6K_yIMuGFUk8X_L9A9_jRT6aq4_'
+        partId: 'H5CBGYjThrMSmaYvRaa5FVKJIzk_'
+        name: Music video clip
+        sourceLayer: Video Clip
+        actionType:
+          - name: pvw
+            label: Preview
+        publicData:
+          fileName: MV000123.mxf
+  globalAdLib:
+    allOf:
+      - $ref: '#/$defs/baseAdLib'
+      - type: object
+        properties:
+          tags:
+            description: Tags attached to this AdLib
+            type: array
+            items:
+              type: string
+    examples:
+      - id: 'C6K_yIMuGFUk8X_L9A9_jRT6aq4_'
+        name: Music video clip
+        sourceLayer: Video Clip
+        actionType:
+          - name: pvw
+            label: Preview
+        tags: ['music_video']
+        publicData:
+          fileName: MV000123.mxf
+  adLibAction:
+    allOf:
+      - $ref: '#/$defs/baseAdLib'
+      - type: object
+        properties:
+          partId:
+            description: Unique id of the part which owns this AdLib
+            type: string
+          tags:
+            description: Tags attached to this AdLib
+            type: array
+            items:
+              type: string
```

Comment:
Is this `adLibAction` referenced anywhere?

I think also the idea in LiveStatusGateway is to homogenize AdLib Pieces and AdLib Actions into a single "AdLib" type, so that the external system doesn't have to understand the difference. This is exemplified by the optional `actionType` property on the `adLib` type/mix-in (action type being an Action thing, but not a Piece thing).

I think this should just be two types: a global AdLib (without the `partId` property) and a regular AdLib (with the `partId` info).